### ISSUE TITLE
[TableNode] Pull-to-refresh

### ISSUE THREAD

**Issue by zdnk** [alexdrone/Render#67]
So I was playing with the TableNode this whole day and did not manage to add UIRefreshControl so it stays there. Do you think you could implement this feature with TableNode?

**Comment by alexdrone** [comment]
I can easily add a `UIRefreshControl` in my example.

` 
    let tableView = TableNode(key: "table_cards", in: self) { view, layout, size in
      if #available(iOS 10.0, *), view.refreshControl == nil {
        view.refreshControl = UIRefreshControl()
      }
      layout.width = size.width
      layout.height = size.height
    }.add(children: cells)

`
I added it in the demo project.

**Comment by zdnk** [comment]
Where that ’.refreshControl’ property coming from on ’UITableView’?

**Comment by alexdrone** [comment]
ios10 apis :)

**Comment by zdnk** [comment]
Oops. Stupid me. Well, I suppose I will have to persuade our PO to drop iOS 9 support 😂

**Comment by zdnk** [comment]
It refreshControl sometimes Just disappears tho :(

**Comment by alexdrone** [comment]
I noticed that as well - will try to fix the issue :)

**Comment by alexdrone** [comment]
Ok that is simply because UITableView doesn't hold onto the refresh control.

you can solve it by doing so 

```
class TableNodeExampleComponentView: ComponentView<TableNodeExampleState> {

  var refreshControl: UIRefreshControl?

  override func render() -> NodeType {
     ....
    // TableNode wraps a 'UITableView' and implements its children through a datasource
    // with cell reuse.
    let tableView = TableNode(key: "table_cards", in: self) { view, layout, size in
      view.backgroundColor = Color.black
      if #available(iOS 10.0, *), view.refreshControl == nil {
        self.refreshControl = UIRefreshControl()
        view.refreshControl = self.refreshControl
      }
      layout.width = size.width
      layout.height = size.height - 64
    }
    ....
  }
```

**Comment by zdnk** [comment]
Well I still have issue with this that when state changes, the UIRefreshControl is simply gone. Or not when states changes, but at some point it will just disappear, completely.
